File: PhaseGate/BackToSender/Python/back_to_sender_logistics.py
def get_rider_wages_per_day_first_stage(first_stage_rate):

    rider_wages = 0
    if first_stage_rate < 50:
        rider_wages = first_stage_rate * 160 + 5000
        
    return rider_wages
    
def get_rider_wages_per_day_second_stage(second_stage_rate):
    
    rider_wages_second_stage = 0
    if second_stage_rate <= 59:
        rider_wages_second_stage = second_stage_rate * 200 + 5000
        
    return rider_wages_second_stage 
    
def get_rider_wages_per_day_third_stage(third_stage_rate):

    rider_wages_third_stage = 0 
    if third_stage_rate <= 69:
        rider_wages_third_stage = third_stage_rate * 250 + 5000
    return rider_wages_third_stage        
          
    
def get_rider_wages_per_day_fourth_stage(fouth_stage_rate) :

    rider_wages_fourth_stage = 0
    if fouth_stage_rate >= 70:
        rider_wages_fourth_stage = fouth_stage_rate * 500 + 5000
    
    
    return rider_wages_fourth_stage         
                  
        
        
        

File: PhaseGate/BackToSender/BackToSenderLogistics.java
import java.util.Scanner;
public class BackToSenderLogistics{

    public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    
    System.out.println("            Back To Sender Logistics Services.");
    
    System.out.println("""
    
        Collection Rate     Amount Per Percel   Base Pay
        Less than 50%                   160       5,000
        50-59%                          200       5,000
        60-69%                          250       5,000
        >= 70%                          500       5,000
       
    """);
    
    System.out.println("Enter your collection rate");
    int userInput = input.nextInt();
    
     if(userInput < 0 || userInput > 100){
        System.out.println("invalid input. Collecetion rate must be between  0 and 100");
    }  
    else if(userInput < 50){
        System.out.println("Your daily wage is:  " + getRiderWagesPerDayFirstStage(userInput));
    }
    else if(userInput <= 59){
        System.out.println("Your daily wage is: " + getRiderWagesPerDaySecondStage(userInput));
        
    }
    else if(userInput <= 69){
        System.out.println("Your daily wage is: " + getRiderWagesPerDayThirdStage(userInput));
        
    }
    else if(userInput >= 70){
        System.out.println("Your daily wage is: " + getRiderWagesPerDayFourthStage(userInput));
            }
   
      
      
    }
    
   public static int  getRiderWagesPerDayFirstStage(int firstStageRate){
   
    int riderWages = 0;
    
    if(firstStageRate < 50){
        riderWages = firstStageRate * 160 + 5000;
        
         }
    return riderWages;
   
   }
   
   public static int getRiderWagesPerDaySecondStage(int secondStageRate){
   
     int riderWagesSecondStage = 0;
   
    if(secondStageRate <= 59){
        riderWagesSecondStage = secondStageRate * 200 + 5000;
    
         }
         return riderWagesSecondStage;
   
   }
   
   public static int getRiderWagesPerDayThirdStage(int thirdStageRate){
    
    int riderWagesThirdStage = 0;
    
    if(thirdStageRate <= 69){
        riderWagesThirdStage = thirdStageRate * 250 + 5000;
          
         }
         return riderWagesThirdStage;
   }
   
   public static int getRiderWagesPerDayFourthStage(int fourthStageRate){
    
    int riderWagesFourthStage = 0;
    
    if(fourthStageRate >= 70 ){
        riderWagesFourthStage = fourthStageRate * 500 + 5000;
        
         }
         return riderWagesFourthStage;
   }


}
